arangod/Cache/Cache.cpp

```diff
@@ -401,45 +405,36 @@ bool Cache::reportInsert(bool hadEviction) {
 }
 
 void Cache::ensureFindStats() {
-  // the happy path is that we already have find stats ready
-  if (ADB_UNLIKELY(!_haveFindStats.load(std::memory_order_relaxed))) {
-    // if we want a single thread to be responsible to create them
-    {
-      std::unique_lock<std::mutex> lock(_findStatsLock);
-      if (!_haveFindStats.load(std::memory_order_relaxed)) {
-        TRI_ASSERT(_findStats == nullptr);
-        _findStats = std::make_unique<FindStats>();
-        if (_enableWindowedStats) {
-          _findStats->findStats = std::make_unique<StatBuffer>(
-              _manager->sharedPRNG(), Manager::kFindStatsCapacity);
-        }
-        _haveFindStats.store(true);
-        lock.unlock();
-        _manager->adjustGlobalAllocation(
-            sizeof(decltype(_findStats)::element_type) +
-            (_enableWindowedStats ? _findStats->findStats->memoryUsage() : 0));
-      }
+  absl::call_once(_findStatsOnceFlag, [this]() {
+    TRI_ASSERT(!_findStatsCreated.load(std::memory_order_relaxed));
+    TRI_ASSERT(_findStats == nullptr);
+
+    _findStats = std::make_unique<FindStats>();
+    if (_enableWindowedStats) {
+      _findStats->findStats = std::make_unique<StatBuffer>(
+          _manager->sharedPRNG(), Manager::kFindStatsCapacity);
     }
-  }
+    _manager->adjustGlobalAllocation(
+        sizeof(decltype(_findStats)::element_type) +
+        (_enableWindowedStats ? _findStats->findStats->memoryUsage() : 0));
+
+    _findStatsCreated.store(true);
```

Comment:
release is enough.
I prefer release other seq_cst, because it better for x86.
For aarch64 is no difference